Source/Wit/Private/Voice/Capture/Emulation/VoiceCaptureEmulation.cpp

```diff
@@ -51,8 +55,40 @@ bool FVoiceCaptureEmulation::Start()
 
 	if (SoundWave != nullptr)
 	{
-		UE_LOG(LogWit, Verbose, TEXT("FVoiceCaptureEmulation: starting with soundwave with duration (%f) and element count (%llu)"), SoundWave->GetDuration(), SoundWave->RawData.GetElementCount());
-		
+		UE_LOG(LogWit, Verbose, TEXT("FVoiceCaptureEmulation: starting with soundwave with duration (%f), element count (%llu), RawPCMDataSize (%lu), bDecompressedFromOgg (%s), ResourceSize (%lu)"),
+			SoundWave->GetDuration(), SoundWave->RawData.GetElementCount(), SoundWave->RawPCMDataSize, SoundWave->bDecompressedFromOgg? TEXT("yes"):TEXT("no") ,SoundWave->ResourceSize);
+
+        if (SoundWave->RawData.GetElementCount() == 0)
+        {
+			bIsPackagedBuild = true;
+			
+			FAudioDevice * AudioDevice = GEngine ? GEngine->GetMainAudioDeviceRaw() : nullptr;
+
+			if (!AudioDevice)
+            {
+				return false;
+			}
+
+			if (!SoundWave)
```

Comment:
Nit: there's a number of formatting issues here. Also by convention we normally don't use (!AudioDevice) for pointers but use the more modern (AudioDevice == nullptr)